long_question_xblock/static/js/src/long_question_xblock.js

```diff
@@ -18,51 +17,39 @@ function StaffGradedAssignmentXBlock(runtime, element) {
             var content = $(element).find('#sga-content').html(template(state));
             var form = $(element).find("#student-answer-form");
 
-            ClassicEditor.create( document.querySelector( '#student-answer-textarea' ) )
-            .then( editor => {
-                if (state.uploaded) {
-                    editor.setData(state.uploaded.student_answer);
-                }
-            })
-            .catch( error => {
-                console.error( error );
-            });
+            if (form.length) {
+                var uploaded = state.uploaded;
+                var student_answer = uploaded ? uploaded.student_answer: '';
+                CKEDITOR.replace("assignment_answer").setData(student_answer);
+            }
 
             $(content).find('.finalize-upload').on('click', function() {
```

Comment:
would be nice for context understanding to rename css classes, `find('.finalize-upload')` depicts out of context meaning here.